src/txdb.cpp

```diff
@@ -30,8 +30,8 @@ static const char DB_FLAG = 'F';
 static const char DB_REINDEX_FLAG = 'R';
 static const char DB_LAST_BLOCK = 'l';
 
-// to distinguish best block for a specific DB type, values correspond to enum vaue (blockdb_wrapper.h)
-static const char DB_BEST_BLOCK_BLOCKDB = 'D';
+// to distinguish best block for a specific DB type, values correspond to enum value (blockdb_wrapper.h)
+static const std::string DB_BEST_BLOCK_BLOCKDB = "1";
 
```

Comment:
not sure i understand the need for this change...all the other databases use the same static const char ?

Reply:
id like this to be the enum value so in the future when more dbs are added we can simply cast the enum value to a string to get the DB_BEST_BLOCK for that database type instead of managing a bunch of characters. it is a string instead of char so we can have a value higher than 9 if needed